.infrastructure/55_cognito.tf

```diff
@@ -354,7 +354,7 @@ data "aws_iam_policy_document" "authenticated_users_policy" {
     actions = ["sts:AssumeRoleWithWebIdentity"]
 
     principals {
-      type        = "Service"
+      type        = "Federated"
```

Comment:
I thought `Service` was the right option, [according to the documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/data-sources/iam_policy_document#identifiers) but apparently it's not 😄